image/image.go

```diff
@@ -702,6 +753,18 @@ func setupSeed(tsto *ToolingStore, model *asserts.Model, opts *Options, local *l
 	return nil
 }
 
+func setRecoveryBootvars(recoverySystemName string) error {
+	// HACK for recovery
+	loader, err := bootloader.Find()
+	if err != nil {
+		return err
+	}
+	return loader.SetBootVars(map[string]string{
+		"snap_mode":            "install",
```

Comment:
shouldn't this be recovery ?

Reply:
Maybe, I was thinking that on a first install we should default to "install" mode, i.e. create writable, create new FDE key, install the system etc. But its all not super well defined yet so we can talk about naming/options etc.